### PR TITLE
Set random state in train test split

### PR DESCRIPTION
**Pull request by sanch7**
Added random state parameter in iterative_train_test_split

### CODE REVIEW

File: skmultilearn/model_selection/iterative_stratification.py
Comment (ChristianSch) [inline skmultilearn/model_selection/iterative_stratification.py]: I think the seed isn't necessarily of type int. it can be one of `None | int | instance of RandomState`, but I'm not 100% sure. Someone should chime in as well.
```suggestion
    random_state : None | int | instance of RandomState
```

**Comment by AlJohri** [comment]
hey @sanch7, just bumping this. thanks!

**Comment by sanch7** [comment]
Thanks for the bump. I have updated the code in my fork as suggested by @ChristianSch , but I don't know how to update this pull request. Do I create another separate request?

**Comment by ChristianSch** [comment]
Seems you got the commit into this PR. This is the way to go imho. The code looks good to me, but I'm a bit hesitant to just merge things I don't understand the background of. So I'll just leave this to @niedakh.

Thanks for your work @sanch7!

**Comment by bhdrozgn** [comment]
Why this is still not merged?

**Comment by sanch7** [comment]
It looks like only people with write access can merge.

**Review by jianlins — COMMENTED** [review]
This is very helpful

**Review by bhdrozgn — APPROVED** [review]
This is a must

**Comment by sanch7** [comment]
Asking @ChristianSch or @niedakh for review!

**Comment by ChristianSch** [comment]
From my side this can be merged. I don't know if this would be released, I'm not very certain I'd consider this project actively maintained atm.

**Pull request merged**
